### ISSUE TITLE
Tandem Fade Overline / Subhead Light Mode Color Contrast

### ISSUE THREAD

**Issue by tommyneu** [unl/wdntemplates#1797]
Tandem fade's overline and subhead have insufficient color contrast in light mode

Text Color: `.unl-dark-gray`
Background Color: `.--bg-lighter-gray`

**Comment by tommyneu** [comment]
Wrong Repo, this was moved to unlcms/project-herbie. Here is a link to the issue [https://github.com/unlcms/project-herbie/issues/955](https://github.com/unlcms/project-herbie/issues/955). Closing this issue
